get.sh

```diff
@@ -12,19 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-echo 'Get binary openjdk...'
-cd $1
-mkdir openjdkbinary
-cd openjdkbinary
-download_url=`curl https://api.github.com/repos/AdoptOpenJDK/openjdk-releases/releases/latest | jq -r '.assets[] | select(.browser_download_url | contains("x64_Linux_jdk")) | {url: .browser_download_url} | select (.url | contains("tar.gz")) | .[]'`
-wget "${download_url}"
-#wget https://github.com/AdoptOpenJDK/openjdk-releases/releases/download/jdk8u152-b01/OpenJDK8_x64_Linux_jdk8u152-b01.tar.gz
-jar_file_url=$download_url
-jar_file_name=${jar_file_url##*/}
-tar -zxvf $jar_file_name
-#tar -zxvf OpenJDK8_x64_Linux_jdk8u152-b01.tar.gz
+openjdktest=$(pwd)
+if [ "$#" -eq 2 ]
+then
+	openjdktest=$2
+	echo 'Get binary openjdk...'
+	cd $1
+	mkdir openjdkbinary
+	cd openjdkbinary
+	download_url=`curl https://api.github.com/repos/AdoptOpenJDK/openjdk-releases/releases/latest | jq -r '.assets[] | select(.browser_download_url | contains("x64_Linux_jdk")) | {url: .browser_download_url} | select (.url | contains("tar.gz")) | .[]'`
+	wget "${download_url}"
+	#wget https://github.com/AdoptOpenJDK/openjdk-releases/releases/download/jdk8u152-b01/OpenJDK8_x64_Linux_jdk8u152-b01.tar.gz
```

Comment:
I am presuming you are leaving this comment in as a convenience, in case folks want to run this script on a machine that doesn't have jq installed.